ow_plexil/src/plans/GoalErrorMonitor.plp

```diff
@@ -0,0 +1,16 @@
+// The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+// Research and Simulation can be found in README.md in the root directory of
+// this repository.
+
+// Runs ClearGoalErrors until a flag to terminate this plan is set.
+
+#include "ow-interface.h"
+
+GoalErrorMonitor:
+{
+  In Boolean continue;
+
+  Exit !continue;
```

Comment:
I recall you recently removing all Exit conditions from Plexil plans. Should this instead be a Repeat condition instead?

Reply:
The only problematic case was Exit combined with Start condition, but that issue has been resolved.  Exit Conditions are fine to use anywhere now.